pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_controller.go

```diff
@@ -154,13 +158,13 @@ func (c *APIReconciler) enqueueAPIResourceSchema(obj interface{}, logger logr.Lo
 	}
 
 	if len(exports) == 0 {
-		logger.V(2).Info("No kubernetes APIExport found for APIResourceSchema")
+		logger.V(2).Info("No APIExports found")
 		return
 	}
 
 	for _, export := range exports {
 		logger.WithValues("apiexport", export.Name).V(2).Info("Queueing APIExport for APIResourceSchema")
-		c.enqueueAPIExport(obj, logger.WithValues("reason", "APIResourceSchema change", "apiResourceSchema", name))
+		c.enqueueAPIExport(export, logger.WithValues("reason", "APIResourceSchema change", "apiResourceSchema", name))
```

Comment:
✨

Reply:
Could the input to enqueueAPIExport be typed maybe?

Reply:
That could be a good thing too 😄